roles/ansible/tower/config-ansible-tower-ocp-ldap-ca/README.md

```diff
@@ -0,0 +1,41 @@
+config-ansible-tower-ocp-ldap-ca
+============================
+
+This role is a helper for `config-ansible-tower-ocp` to create and provision CA certificate secret and ConfigMap which is  used to validate connection with LDAP
+
+## Requirements
+
+  -  running OpenShift Cluster
+  -  'oc' client installed on the Ansible host
+
+
+## Role Variables
+
+The variables used to create and deploy CA secret/ConfigMap  Ansible Tower on OpenShift are outlined in the table below. 
+
+| Variable | Description | Required | Defaults |
+|:---------|:------------|:---------|:---------|
+|openshift_project|OCP project in which Ansible Tower is deployed|yes||
+|ansible_tower.ldap.ca_cert| Path to CA pem file to be uploaded to Ansible Tower-file should have the "ldap.pem" name|yes|
+|ansible_tower.ldap.ldap_py| Path to ldap bootstrap file for Ansible Tower|yes|
```

Comment:
Do we need to have the `ldap.py` configurable? I'd say probably not. At least we should make it optional as the file is already supplied under `files` and that's the one we should use (anything else would be an "advanced" use-case).